Sources/Run/Commands.swift

```diff
@@ -49,10 +52,45 @@ final class CreateSessionCommand: Command, ConfigInitializable {
     }
 }
 
+final class TestPayments: Command, ConfigInitializable {
+    public let id = "test_payments"
+    public let help = ["This command does things, like foo, and bar."]
+    public let console: ConsoleProtocol
+    private let fbId = fbIdLilia
+    
+    public init(console: ConsoleProtocol) {
+        self.console = console
+    }
+    
+    public convenience init(config: Config) throws {
+        let console = try config.resolveConsole()
+        self.init(console: console)
+    }
+    
+    
+    public func run(arguments: [String]) throws {
+        let price = "30"
+        let spot = 2
+        
+        drop.send(message: "Tailor-made-jane has chosen you to be on the next show!", senderId: fbId, messagingType: .NON_PROMOTIONAL_SUBSCRIPTION)
+        drop.send(message: "There are " + spot.string + " spots left to be on the show", senderId: fbId, messagingType: .NON_PROMOTIONAL_SUBSCRIPTION)
```

Comment:
p2 style: split to few lines. You can use commnad + A for format code. See the next same comment